### DISCUSSION TITLE
The fate of Email/Password authentication

### DISCUSSION THREAD

**Discussion by acheong08** [acheong08/ChatGPT#1237] [Polls]
- #1236

**Comment by Germey** [comment]
I think suspending the support of email/password auth is not a good practice, just because of privacy concern. 

Actually, one user can register many ChatGPT accounts based on many "unusual" ways, and these account's email/password does not realted to any user's privacy info. So user actually not care TOO MUCH about leaking the email/password. If user cares, they can choose another way to authenticate instead of using email/password, we can put the Privacy Notice to inform user about it.

**Comment by fkxxyz** [comment]
I'm all for making a 3rd party authentication API. I have a large number of OpenAI accounts and I'm not really worried about security, I just want a convenient way to manage and log into them. Just remind users of the privacy policy

**Reply by acheong08** [reply]
Thank you for the feedback. I'll make that option available soon.

**Reply by acheong08** [reply]
I will need a huge number of proxies. There is IP based rate limits on auth
